doc/reference/config-options-launch.rst

```diff
@@ -0,0 +1,103 @@
+.. _launch-config-options:
+
+Launch Configuration Options
+##############################
+
+As explained in :ref:`acrn_configuration_tool`, launch configuration files
+define post-launched User VM settings. This document describes these option settings.
+
+``uos``:
+  Specify the User VM with its relative ID to Service VM by the ``id``
+  attribute.
+
+``uos_type``:
```

Comment:
I've got a message to engineering asking about these, since **none** of these values are used in the current XML files nor in the Python code that processes them.  Instead, we're using names such as KATA_VM, POST_LAUNCHED_VM, and PRE_RT_VM.  Some explaining is in order.